raiden/raiden_service.py

```diff
@@ -272,16 +269,34 @@ def __init__(
         self.gas_reserve_lock = gevent.lock.Semaphore()
         self.payment_identifier_lock = gevent.lock.Semaphore()
 
+        # Flag used to skip the processing of all Raiden events during the
+        # startup.
+        #
+        # Rationale: At the startup, the latest snapshot is restored and all
+        # state changes which are not 'part' of it are applied, the criteria to
+        # re-apply the state changes is their 'absence' in the snapshot, /not/
+        # their completeness. Because these state changes are re-execute
```

Comment:
s/re-execute/re-exected

Reply:
I have already fixed this and other typos :P.

But not re-exected!